app/controllers/setup_controller.rb

```diff
@@ -67,7 +67,8 @@ def configure
   def worker_bootstrap
     @controller_node = Pillar.value pillar: :dashboard
 
```

Comment:
just add this around the method definidion:
```ruby
# rubocop:disable Metrics/CyclomaticComplexity
def worker_bootstrap
  ...
end
# rubocop:enable Metrics/CyclomaticComplexity
```

Reply:
done